R/digest.R

```diff
@@ -19,8 +19,8 @@
 ##  along with digest.  If not, see <http://www.gnu.org/licenses/>.
 
 
-digest <- function(object, algo=c("md5", "sha1", "crc32", "sha256", "sha512",
-                                  "xxhash32", "xxhash64", "murmur32"),
+digest <- function(object, algo=c("md5", "sha1", "crc32", "sha256", "sha256_new", "sha512",
```

Comment:
Please remove addition of sha256_new in the middle.  New ones go to the end.  Do we need `sha256_new` ?